pkg/ride/environment.go

```diff
@@ -1106,7 +1107,7 @@ func (e *EvaluationEnvironment) ChooseSizeCheck(v ast.LibraryVersion) {
 	e.setLibVersion(v)
 	if v > ast.LibV2 {
 		e.check = func(l int) bool {
-			return l <= maxMessageLength
+			return l <= proto.MaxDataWithProofsBytes
```

Comment:
These are test functions, can't be used in production code. I thought about removing those `v2check` and `v3check` functions because they are not helping the environment functionality.

Reply:
Hmm, okay. If you want to remove them, then we'll consider it in a next PR. Marking as resolved.